nav2_simple_commander/nav2_simple_commander/robot_navigator.py

```diff
@@ -112,7 +112,7 @@ def setInitialPose(self, initial_pose):
         self.initial_pose = initial_pose
         self._setInitialPose()
 
-    def goThroughPoses(self, poses, behavior_tree=''):
+    def goThroughPoses(self, poses, behavior_tree='', planner_id=''):
```

Comment:
@mjcarroll am I correct in saying for python that I cannot backport this due to ABI stability policies, even with a default argument making it optional? I've never been 100% clear how serious that was for an interpreted language like Python3.